examples/chatmemberbot.py

```diff
@@ -14,6 +14,19 @@
 import logging
 from typing import Optional, Tuple
 
+import telegram
+
+try:
+    from telegram import __version_info__
+except ImportError:
+    __version_info__ = (0, 0, 0, 0, 0)  # type: ignore[assignment]
+
+if __version_info__ < (20, 0, 0, "alpha", 0):
+    tg_ver = telegram.__version__
```

Comment:
```suggestion
    from telegram import __version__ as tg_ver
```
and remove the import above